example/models_directory/configuration_interaction.json

```diff
@@ -0,0 +1,34 @@
+{
+    "tier1": {
+        "name": "physics-based",
+        "slug": "pb"
+    },
+    "tier2": {
+        "name": "quantum-mechanical",
+        "slug": "qm"
+    },
+    "tier3": {
+        "name": "ab initio",
+        "slug": "abini"
+    },
+    "type": {
+        "name": "configuration interaction",
+        "slug": "ci"
+    },
+    "subtype": {
+        "name": "MR-CISD",
+        "slug": "mrcisd"
+    },
+    "modifier": [
+        "multi-reference"
+    ],
+    "tags": [
+        "wavefunction",
+        "scaling-degree:6",
+        "excited-states",
+        "user-adjustable"
+    ],
+    "method": {
+        "...": "include(../method.json)"
+    }
+}
```

Comment:
Shall we include the folder structure for the model hierarchy inside `model_directory`?